code/framework/api/src/main/java/io/cattle/platform/api/servlet/ApiRequestFilter.java

```diff
@@ -37,6 +37,9 @@
     private static final DynamicStringListProperty IGNORE = ArchaiusUtil.getList("api.ignore.paths");
     private static final DynamicStringProperty PL_SETTING = ArchaiusUtil.getString("ui.pl");
     private static final String PL = "PL";
+    private static final String LOCALIZATION = System.getenv("CATTLE_LOCALIZATION");
+    private static final String LOCALIZATION_CHINESE = "zh-Hans-CN";
```

Comment:
This is unneeded per https://github.com/rancher/cattle/pull/1994#issuecomment-248626932